packages/appsync-emulator-serverless/serverCore.js

```diff
@@ -95,6 +95,33 @@ class SubscriptionServer {
         continue;
       }
 
+      let shouldPublish = true;
+
+      // subscription filter based on variables
+      // variable key/value pair must match payload key/value pair
```

Comment:
is this the entire [spec](https://github.com/graphql/graphql-spec/blob/master/rfcs/Subscriptions.md) ?

Reply:
Based my implementation on [this](https://docs.aws.amazon.com/appsync/latest/devguide/real-time-data.html)

See `Using Subscription Arguments` heading